bin/docker.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+CURRDIR=$(dirname "${0}")
+source ${CURRDIR}/common.sh
+
+: ${DOCKER_RUN:="docker run \
```

Comment:
If it's not set, set it. This way I could override this variable from a script that wraps this one (as I actually do in the `cli/bin/docker.sh` script.